stable/dex-controller/templates/certificates.yaml

```diff
@@ -1,4 +1,8 @@
+{{- if  (.Capabilities.APIVersions.Has "certmanager.k8s.io/v1") }}
+apiVersion: certmanager.k8s.io/v1
```

Comment:
whoops, almost missed this:

```suggestion
apiVersion: cert-manager.k8s.io/v1
```

Reply:
this was the reason i wanted to self merge, i figured this was wrong, anyway closing this since its the wrong repo.